addons/mail/models/mail_alias_domain.py

```diff
@@ -183,14 +183,17 @@ def _find_aliases(self, email_list):
         or default from) and mail aliases from an email list. """
```

Comment:
While we are here, maybe we should add a docstring for `email_list`
```suggestion
        or default from) and mail aliases from an email list.

        :param email_list: list of normalized emails; normalization / removing
            wrong emails is considered as being caller's job
        """
```